clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Job.java

```diff
@@ -146,6 +148,17 @@ public Job(
     iamProfile = grpcJob.getJobDescriptor().getContainer().getSecurityProfile().getIamRole();
     allocateIpAddress = true;
     submittedAt = new Date(grpcJob.getStatus().getTimestamp());
+    constraints = new SubmitJobRequest.Constraints();
+    if (grpcJob.getJobDescriptor().getContainer().getHardConstraints().getConstraintsMap()
```

Comment:
nit: prob be nicer to have a "temp" var:
```
constraintsMap = grpcJob.getJobDescriptor().getContainer().getHardConstraints().getConstraintsMap()
if (constraintsMap != null) {
    ....setHard(constraintsMap);
}
```

Reply:
👍